services/movie.py

```diff
@@ -0,0 +1,50 @@
+from db.models import Movie, Genre, Actor
+from typing import List, Optional
+
+
+def get_movies(
+    genres_ids: Optional[List[int]] = None,
+    actors_ids: Optional[List[int]] = None
+) -> List[Movie]:
+    movies = Movie.objects.all()
```

Comment:
Code Efficiency: Consider filtering the movies only when necessary, instead of loading all movies initially. This will improve performance by avoiding unnecessary database queries.